ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -0,0 +1,633 @@
+      (key) => key.toLowerCase() === address.toLowerCase(),
+    );
+
+    let standard;
+    if (addressIsValid) {
+      try {
+        ({ standard } = await getTokenStandardAndDetails(
+          standardAddress,
+          selectedAddress,
+          null,
+        ));
+      } catch (error) {
+        // ignore
+      }
+    }
+
+    const addressIsEmpty = address.length === 0 || address === EMPTY_ADDRESS;
+
+    switch (true) {
+      case !addressIsValid && !addressIsEmpty:
+        setCustomAddressError(t('invalidAddress'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case standard === TokenStandard.ERC1155 ||
+        standard === TokenStandard.ERC721:
+        setNftAddressError(
+          t('nftAddressError', [
+            <ButtonLink
+              className="import-tokens-modal__nft-address-error-link"
+              onClick={() => {
+                dispatch(showImportNftsModal());
+                onClose();
+              }}
+              color={TextColor.primaryDefault}
+              key="nftAddressError"
+            >
+              {t('importNFTPage')}
+            </ButtonLink>,
+          ]),
+        );
+        break;
+
+      case isMainnetToken && !isMainnet:
+        setMainnetTokenWarning(t('mainnetToken'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case Boolean(identities[standardAddress]):
+        setCustomAddressError(t('personalAddressDetected'));
+        break;
+
+      case checkExistingAddresses(address, tokens):
+        setCustomAddressError(t('tokenAlreadyAdded'));
+        break;
+
+      default:
+        if (!addressIsEmpty) {
+          attemptToAutoFillTokenParams(address);
+          if (standard) {
+            setTokenStandard(standard);
+          }
+        }
+    }
+  };
+
+  // Determines whether to show the Search/Import or Confirm action
+  const isConfirming = mode === 'confirm';
+
+  return (
+    <Modal
+      isOpen
+      onClose={() => {
+        dispatch(clearPendingTokens());
+        onClose();
+      }}
+      className="import-tokens-modal"
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader
+          onBack={isConfirming ? () => setMode('') : null}
+          onClose={() => {
+            dispatch(clearPendingTokens());
+            onClose();
+          }}
+        >
+          {t('importTokensCamelCase')}
+        </ModalHeader>
+        {isConfirming ? (
+          <ImportTokensModalConfirm
+            onBackClick={() => {
+              dispatch(clearPendingTokens());
+              setMode('');
+            }}
+            onImportClick={async () => {
+              await handleAddTokens();
+              onClose();
+            }}
+          />
+        ) : (
+          <>
+            <Tabs t={t}>
```

Comment:
We need to have a margin-top here. As per design, I think it should be 24px
![Screenshot 2023-08-01 at 10 29 46 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/85d76e75-7a23-4251-8c46-1d589bce426a)